app/styles/atoms/progress-bar.less

```diff
@@ -0,0 +1,41 @@
+.oss-progress-bar--wrapper {
+  min-width: 220px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  p {
+    font-weight: 200;
+    margin: 0 6px;
+  }
+}
+
+.oss-progress-bar {
+  width: 100%;
+  border-radius: var(--border-radius-sm);
+  background-color: darken(@upf-gray-light, 8%);
```

Comment:
We deprecated the use of darken, if the color doesn't exist in the design system, sync with William or set the hex value directly

Reply:
Doesn't exist in DSM , checked with William & set the hex according to the existing color